src/main/java/uk/gov/companieshouse/web/accounts/controller/fullaccounts/FullAccountsCriteriaController.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.companieshouse.web.accounts.controller.fullaccounts;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.servlet.view.UrlBasedViewResolver;
+import uk.gov.companieshouse.web.accounts.controller.BaseController;
+
+@Controller
+@RequestMapping("/accounts/full-accounts-criteria")
+public class FullAccountsCriteriaController extends BaseController{
```

Comment:
Maybe worth renaming this to `GovukFullAccountsCriteriaController` for consistency with the other controllers in this PR?

Reply:
👍